### DISCUSSION TITLE
Ender 3v2 with Sprite Extruder Pro and BL touch - Setting

### DISCUSSION THREAD

**Discussion by b771x** [mriscoc/Ender3V2S1#883] [Q&A]
Hello,
I use firmware Ender3V2-422-BLTUBL-T13-MPC-20230312.bin. I have problem to make correct physical and probe setting. Who has the same HW configuration: Please can You help me:
1. What Mriscoc version of firmware to use
2. What physical and probe exact values to setup ?
Thanks for the answer

**Comment by JohnDelisle** [comment]
I'm running a self-compiled Ender3V2-422-BLTUBL-T13-LA-MPC.  I'm working through configuring it, but used to run your firmware and can share most of the settings for Ender 3v2 with Sprite Pro and CR Touch.

Physical settings:
X bed size: 215
Y bed size: 220
X min pos: 0
Y min pos: -5
X max pos: 215
Y max pos: 220
Z max pos: 235

Bed PID (you should run an auto-tune, but here's mine):
Set Kp: 98.27
Set Ki: 17.55
Set Kd: 366.86
Temp: 60
Pid Cycles: 5

Hotend PID (again - auto-tune, don't use mine, but for reference..):
Set Kp: 14.84
Set Ki: 1.13
Set Kd: 48.67
Temp: 230
Pid Cycles: 5

Probe Settings: 
Probe X Offset: -31.4
Probe Y Offset: -40.1
Probe Z Offset: -2.76
Z Feed Rate: 480
Multi Probing: 2
Enable HS Mode: Yes

Mesh Leveling:
Mesh Points: 7x7
Fade Height: 10

Mesh Inset:
Mesh X Min: 10
Mesh X Max: 183.6
Mesh Y Min: 10
Mesh Y Max: 179.9

**Reply by ahtinurme** [reply]
Thank you, this works!

**Reply by ShoobyDooWupWup** [reply]
Signed up to GitHub just to say thank you for this. I bought a BambuLab and haven't used my E3 V2 for a while, decided to get it out and play around with it. Apart from the Z height (due to the type of bed more than likely) everything was perfect, did the PID's myself. Can't thank you enough for the BL Touch settings mate.

**Reply by nikopatel** [reply]
> I'm running a self-compiled Ender3V2-422-BLTUBL-T13-LA-MPC. I'm working through configuring it, but used to run your firmware and can share most of the settings for Ender 3v2 with Sprite Pro and CR Touch.
> 
> Physical settings: X bed size: 215 Y bed size: 220 X min pos: 0 Y min pos: -5 X max pos: 215 Y max pos: 220 Z max pos: 235
> 
> Bed PID (you should run an auto-tune, but here's mine): Set Kp: 98.27 Set Ki: 17.55 Set Kd: 366.86 Temp: 60 Pid Cycles: 5
> 
> Hotend PID (again - auto-tune, don't use mine, but for reference..): Set Kp: 14.84 Set Ki: 1.13 Set Kd: 48.67 Temp: 230 Pid Cycles: 5
> 
> Probe Settings: Probe X Offset: -31.4 Probe Y Offset: -40.1 Probe Z Offset: -2.76 Z Feed Rate: 480 Multi Probing: 2 Enable HS Mode: Yes
> 
> Mesh Leveling: Mesh Points: 7x7 Fade Height: 10
> 
> Mesh Inset: Mesh X Min: 10 Mesh X Max: 183.6 Mesh Y Min: 10 Mesh Y Max: 179.9

Hi Awesome Helpful post. 
could you please tell me if this settings (offcourse not the bedsize) will work for Ender 3/Ender 3 Pro on BTT SKR mini e3 V3.0 board? 
and how to tune for PID for my board?

**Comment by b771x** [comment]
Thank you very much for detail answer I will set them and we will see.

Dňa ne 23. 4. 2023 o 21:21 John Delisle ***@***.***>
napísala(a):

**Comment by b771x** [comment]
My printer works well after new settings. Only MPC is maybe not ok. Please
look at photo. Target was 200 but calibration ended on 217. Is it ok?


Dňa po 24. 4. 2023 o 10:46 pribylinecm ***@***.***> napísala(a):

**Comment by b771x** [comment]
[IMG_1333](https://user-images.githubusercontent.com/102349344/234032409-5cfb72cb-55f8-427d-b66f-daf5a75a4d08.JPG)

**Comment by igaldos** [comment]
I posted this a few mins ago. Hopefully this helps!

https://www.reddit.com/r/ender3v2/comments/1i69g2p/ender_3v2_upgrade_guidejourney_in_2025/

**Comment by forbookingitems-design** [comment]
I just upgraded my Ender 3 V2 with Sprite Pro and as per some of the youtube video the recomended firm ware is Ender3V2-422-BLTUBL-T13-MPC-20230312.bin. 
I have a CR Touch and is it true that i do not need to set the Offsets of X and Y when using this? It seems to print in the center though. 
The Auto mesh however does not probe correct in the all the sides.  Is there any work around? so that i would able to mesh across

**Reply by mriscoc** [reply]
Did you follow the calibration guide from the Wiki? There are physical limits to the area that can be reached.

**Reply by forbookingitems-design** [reply]
Had tried those but will recheck those. As you mentioned may be there are some physical limits.
Thank you.
